subprojects/docs/src/docs/release/notes.md

```diff
@@ -136,6 +136,12 @@ Additional options were added for use with `-Dorg.gradle.debug=true`. These allo
 
 See [the documentation](userguide/command_line_interface.html#sec:command_line_debugging) for details.
 
+### Test Suites ###
+
+#### Detect Additional Test Suite Sources in IntelliJ IDEA ####
+
+The [IntelliJ IDEA Plugin](userguide/idea_plugin.html) plugin will now automatically mark any [SourceSets](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.SourceSet.html) created by adding additional Test Suites via the [Test Suite Plugin](userguide/jvm_test_suite_plugin.html) as test source directories within the IDE.
```

Comment:
FYI, we shouldn't link to docs.gradle.org/current in the release notes because they'll go to the wrong version.

```suggestion
The [IntelliJ IDEA Plugin](userguide/idea_plugin.html) plugin will now automatically mark all source directories used by a [JVM Test Suite](userguide/jvm_test_suite_plugin.html#declare_an_additional_test_suite) as test source directories within the IDE. 

The [JVM Test Suite Plugin](userguide/jvm_test_suite_plugin.html) is an incubating plugin that makes it easier to create additional sets of tests in a Java project.

The Eclipse plugin will be updated in a future version of Gradle.
```